### PR TITLE
Introduce trivy to scan images

### PR DESCRIPTION
**Pull request by hanahmily**
As the title indicated, the change introduces trivy action to scan images.

### CODE REVIEW

File: .github/workflows/docker-ci.yaml [hunk 1]
```diff
@@ -59,6 +59,24 @@ jobs:
             ${{ runner.os }}-maven-
       - name: Build docker image
         run: export ES_VERSION=${{ matrix.es }} && export TAG=${{ matrix.es }} && (make docker || make docker)
+      - name: Scan OAP image
+        uses: aquasecurity/trivy-action@master
```
Comment (wu-sheng) [inline .github/workflows/docker-ci.yaml:63]: Should we lock the version/commit id? Depending on master would be unstable

Comment (hanahmily) [reply .github/workflows/docker-ci.yaml:63]: make sense. I will pick up a sha1 later

File: oap-server/pom.xml [hunk 2]
```diff
@@ -79,8 +79,8 @@
         <commons-lang3.version>3.7</commons-lang3.version>
         <commons-text.version>1.4</commons-text.version>
         <simpleclient.version>0.6.0</simpleclient.version>
-        <apollo.version>1.4.0</apollo.version>
-        <nacos.version>1.3.1</nacos.version>
+        <apollo.version>1.8.0</apollo.version>
+        <nacos.version>1.4.2</nacos.version>
```
Comment (wu-sheng) [inline oap-server/pom.xml:83]: This kind of changes causes LICENSE update required.

Comment (hanahmily) [reply oap-server/pom.xml:83]: done

File: apm-webapp/pom.xml [hunk 3]
```diff
@@ -101,6 +104,26 @@
...
+            <groupId>org.apache.tomcat.embed</groupId>
+            <artifactId>tomcat-embed-core</artifactId>
+            <version>${tomcat.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
+            <artifactId>tomcat-embed-websocket</artifactId>
+            <version>${tomcat.version}</version>
+        </dependency>
         <dependency>
```
[the remark this answers is not in the document]
Comment (hanahmily) [reply apm-webapp/pom.xml:127]: ignore

File: oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml [hunk 4]
```diff
@@ -41,7 +41,7 @@
         <dependency>
```
[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

[the remark this answers is not in the document]
Comment (hanahmily) [reply oap-server/server-cluster-plugin/cluster-consul-plugin/pom.xml:41]: ignore

File: apm-webapp/pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (hanahmily) [reply apm-webapp/pom.xml:127]: ignore

**Comment by hanahmily** [comment]
This violates the CVE policy of Apache Foundation

**Pull request closed without merging**